grakn-graph/src/main/java/ai/grakn/graph/internal/AbstractGraknGraph.java

```diff
@@ -642,7 +643,7 @@ void validateGraph() throws GraknValidationException {
         }
     }
 
-    private void submitCommitLogs(Map<Schema.BaseType, Set<String>> concepts){
+    protected void submitCommitLogs(Map<Schema.BaseType, Set<String>> concepts){
```

Comment:
Why is this protected now ? Is it being overridden or used somewhere outside this class ?